linux_os/guide/services/ldap/389_ds/package_389-ds-base_removed/rule.yml

```diff
@@ -0,0 +1,52 @@
+documentation_complete: true
+  
+prodtype: rhel6,rhel7,rhel8
+
+title: 'Uninstall 389-ds-base Package'
+
+description: |-
+    The <tt>389-ds-base</tt> package should be removed if not in use.
+    Is this system the 389-ds server? If not, remove the package.
```

Comment:
This line seems to be superfluous.

Reply:
It was lifted from the OpenLDAP rule.  I can drop it if you'd like.

Reply:
The same sentiment is expressed later through lines 11-14. Suggest dropping the ``Is the system the ....`` part.

Bonus points if you feel like cleaning up the original OpenLDAP language too! Won't hold up a merge if not.